composer.json

```diff
@@ -27,7 +27,8 @@
     },
     "autoload": {
         "psr-4": {
-            "LaravelEnso\\ActivityLog\\": "src/"
+            "LaravelEnso\\ActivityLog\\": "src/",
+            "LaravelEnso\\ActivityLog\\Test": "tests/"
```

Comment:
why do we need this?

Reply:
I added the namespace for tests because preventing from class name collision

Reply:
The class collision can only happen when testing but we can use prefixes for that, right?

I can't see any benefit for autoloading the tests namespaces when env !== 'testing'.

Reply:
your are right, I made a mistake! it can be used in autoload-dev or use prefixes, I fix it and push it again